### PR TITLE
As standalone app

### PR DESCRIPTION
**Pull request by michielbdejong**

### CODE REVIEW

**Comment by michielbdejong** [comment]
@megoth can you review?

File: package.json [hunk 1]
```diff
@@ -3,12 +3,25 @@
...
+    "build-lib": "babel src -d lib --source-maps --extensions '.ts,.js'",
+    "build-types": "tsc --emitDeclarationOnly",
+    "build-version": "./timestamp.sh > src/versionInfo.ts  && eslint 'src/versionInfo.ts' --fix",
     "lint": "eslint '*.js'",
     "lint-fix": "eslint '*.js' --fix",
     "test": "npm run lint",
     "prepublishOnly": "npm test",
-    "postpublish": "git push origin master --follow-tags"
+    "postpublish": "git push origin master --follow-tags",
+    "watch": "webpack --watch",
```
Comment (megoth) [inline package.json:23]: Maybe we should distinguish between watch for the webapp, and watch for the libs? E.g. could have `app-watch` and `lib-watch`?

Comment (michielbdejong) [reply package.json]: Renamed it. Thanks!

**Comment by michielbdejong** [comment]
@timbl This moves files into the src/ folder, so please `git pull` this into your local branch(es).

**Pull request merged**